templates/adherent/dashboard.html.twig

```diff
@@ -0,0 +1,213 @@
+{% extends 'base.html.twig' %}
+
+{% block content %}
+    <section class="dashboard l__wrapper">
+        <div class="dashboard__welcome">
+            {% set imagePath = summary is not null ? path('asset_url', { path: summary.picturePath }) : '/images/user.png' %}
+
+            <p>
+            {% if summary is not null %}
+                <img src="{{ path('asset_url', { path: summary.picturePath }) }}" title="{{ 'dashboard.picture.update'|trans }}">
+            {% else %}
+                <a href="{{ path('app_summary_manager_step', {'step': 'photo'}) }}">
```

Comment:
pls can you remove `'` from the key? `{ step: 'photo' }`

Reply:
done